transaction/core/src/account_keys.rs

```diff
@@ -362,8 +363,24 @@ impl AccountKey {
 
         // Compute fog_authority_sig as a signature over self.fog_authority_key_fingerprint
         if !self.fog_report_url.is_empty() {
-            // FIXME: FOG-106 fog_authority_sig should be a Schnorrkel sig using subaddress_view_private
-            result.fog_authority_sig.extend(&[9u8, 9u8, 9u8, 9u8]);
+            // Construct a MiniSecretKey from the bytes directly, then expand to SecretKey
+            let scalar_bytes = self.subaddress_view_private(index).to_bytes();
+            let mut bytes_slice = [0u8; 32];
+            bytes_slice[..32].copy_from_slice(&scalar_bytes);
+            let mini_secret = MiniSecretKey::from_bytes(&bytes_slice)
```

Comment:
I think MiniSecretKey is a "precursor" that is used to generate a new scalar (i.e. our RistrettoPrivate). See:

https://github.com/w3f/schnorrkel/blob/master/src/keys.rs#L51-L56

and 

See: https://tools.ietf.org/html/rfc8032#section-5.1.5

Here, however, the scalar -- the `subaddress_view_private` key -- already exists. I think you can use that directly to create a SecretKey via `SecretKey::from_bytes(bytes)`, where bytes is a [u8; 64] containing `subaddress_view_private.to_bytes()` and a random 32-byte nonce.

Reply:
I haven't read through the code/comments, but I'm pretty confident that we cannot use our `PrivateKey` as the `SecretKey` directly, we should use it as a `MiniSecretKey`

Reply:
@jcape - we need to follow up on this - created [MC-1611](https://mobilecoin.atlassian.net/browse/MC-1611) to capture changes in our keys crate that need to be made to support using MiniSecretKey

Reply:
In off-line discussions, we agreed that using the SecretKey with the Hash(ristretto_private | message) as the nonce is likely sufficient, and we plan to do a careful review and analysis of this scheme wrt the Scalar also being used and derived through CryptoNote math.